lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java

```diff
@@ -79,7 +80,10 @@ public VerifiableCredentialService(
 
     @Tracing
     public SignedJWT generateSignedVerifiableCredentialJwt(
-            String subject, PersonIdentityDetailed personIdentity, KBVItem kbvItem)
+            String subject,
```

Comment:
Can we remove subject, this b'cos it is the `sessionItem`

Reply:
have removed subject